lib/utils/post_queries.dart

```diff
@@ -47,31 +47,12 @@ class PostQueries {
   ///
   /// **returns**:
   /// * `String`: The query related to gettingPostsbyId
-  String getPostById(String postId) {
+  String getPostCommentsById(String postId) {
     return """
       query {
         post(id: "$postId")
         { 
-          _id
-          text
-          createdAt
-          imageUrl
-          videoUrl
-          title
-          commentCount
-          likeCount
-          creator{
-            _id
-            firstName
-            lastName
-            image
-          }
-          organization{
-            _id
-          }
-          likedBy{
-            _id
-          }
+          _id,
           comments{
```

Comment:
Is there other query which fetch post by ID? or getPostById was only used for comments?

Reply:
@Ayush0Chaudhary please have a look at this [comment](https://github.com/PalisadoesFoundation/talawa/issues/2333#issuecomment-1907678196).